### DISCUSSION TITLE
Artillery requests keep getting error HTTP 499 by NGINX

### DISCUSSION THREAD

**Discussion by lee-pai-long** [artilleryio/artillery#2361] [Artillery]
I keep getting error 499 by nginx when using artillery. The HTTP 499 status code is not a standard HTTP status code, meaning it is not defined in the HTTP/1.1 specification. This status code is used by the nginx web server to indicate that the client closed the connection before the server could send a response.

Why does artillery systematically close connections before response ?

**Comment by hassy** [comment]
What do Artillery logs look like? Is nginx proxying to another service?

**Reply by lee-pai-long** [reply]
@hassy do you mean the logs in debug mode or just the load test result ? Nginx is just proxing one service.
